build-system/azure-pipeline.template.yaml

```diff
@@ -27,10 +27,9 @@ jobs:
         submodules: recursive  # set to 'true' for a single level of submodules or 'recursive' to get submodules of submodules
         persistCredentials: true
       - task: UseDotNet@2
-        displayName: 'Use .NET Core Runtime 3.1.10'
+        displayName: 'Use .NET 6'
```

Comment:
7 :)

Reply:
LOL how on earth did this PR get auto-merged so fast? Whoops.

Reply:
And look at this indenting! https://github.com/akkadotnet/Akka.Persistence.PostgreSql/pull/179/files#diff-4d59c677ea4c9112e0ce2f2e527693f48dcbcf66ba4eeb4b01abb5f3da8bbe11 ;)